src/cq_cam/routers.py

```diff
@@ -58,15 +58,20 @@ def vertical_ramp(
 """
 
 
-def rapid_to(v: cq.Vector, rapid_height: float, safe_plunge_height=None):
+def rapid_to(
+    v: cq.Vector,
+    rapid_height: float,
+    safe_plunge_height=None,
+    feed: Optional[float] = None,
```

Comment:
plunge_feed ?

Reply:
Yeah that's what I meant. I will correct soon!

Reply:
Done